e2e/tests/portal/fixtures.ts

```diff
@@ -0,0 +1,30 @@
+import { test as base, Page } from '@playwright/test';
+export { expect } from '@playwright/test';
+
+export enum UserPrefix {
+  BceidBasic = 'BCEID_BASIC',
+}
+
+interface FixtureOptions {
+  userPrefix: string;
+}
+
+interface Fixtures {
+  inboxLoggedIn: Page;
+}
+
+export const test = base.extend<FixtureOptions & Fixtures>({
```

Comment:
Ideally the name of fixture should represent the action that it performs, in this case it is login

Reply:
It does, on line 18: `inboxLoggedIn`. The way fixtures work in Playwright is that you extend the main test function with an object that contains the fixtures.